Nodejs/Product/Nodejs/Project/NodejsProjectNode.cs

```diff
@@ -304,6 +310,10 @@ public NodejsProjectNode(NodejsProjectPackage package)
                 // enable TypeScript on the project automatically...
                 SetProjectProperty(NodejsConstants.EnableTypeScript, "true");
                 SetProjectProperty(NodejsConstants.TypeScriptSourceMap, "true");
+#if DEV14
+                // Reset cached value, so it will be recalculated later.
+                this.shouldAcquireTypingsAutomatically = null;
```

Comment:
Because we know that `EnableTypeScript` is true, we should just set this to false here.

After than, looks good to me.